### PR TITLE
Last Documents added

### PR DESCRIPTION
**Pull request by RojoRX**
Agregue el componente para mostrar los últimos documentos publicados, no logro encontrar la url de la api que necesito para recibir estos datos, por lo que uso de momento otra api para probar que lo demás funciona correctamente

### CODE REVIEW

File: src/components/LastDocsComponent/LastDocs.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    constructor(props) {
+        super(props);
+        this.state = {
+            items: [],
+            checked: false
+        };
+        
+    }
+    componentDidMount() {
+        fetch('https://fakestoreapi.com/products?limit=5').then(res => res.json()).then(json => {
```
Comment (Ditmar) [inline src/components/LastDocsComponent/LastDocs.jsx]: componentDidMount is using in react with classes, that is, react legacy, in this case, we are using react with a hook if you need to control the fetch you need to put the request, into the useEffect function,

File: src/components/DashboardComponent/Orders.jsx [hunk 2]
```diff
@@ -1,86 +1,78 @@
...
+            <Typography color="text.secondary"
+                sx={{display: 'inline' }}>Group
+                <Typography color="primary"
+                    sx={{display: 'inline', ml:1}}>
+                    Support
+                </Typography>
+            </Typography>
+            <Table size="small" sx={{mt:2, ml:0}}>
+                <TableBody> {
+                    this.state.items.map((row) => (
```
Comment (Ditmar) [inline src/components/DashboardComponent/Orders.jsx]: Please try to separate this large conde in small components

**Review by Ditmar — CHANGES_REQUESTED** [review]
Please use useEffect the fetch function should be in the async directory; use the functions created in https://github.com/Ditmar/plagiarist-g2/blob/master/src/async/api.js to make the request

**Comment by RojoRX** [comment]
Good evening Engineer. I corrected the code problems, now using useEffect and making the code shorter with other components

File: src/components/DashboardComponent/Orders.jsx [hunk 3]
```diff
@@ -1,86 +1,78 @@
...
-  ),
-];
+// let api="http://3.23.86.147/api-doc/#/default/get_server_summary_report";
 
 function preventDefault(event) {
-  event.preventDefault();
+    event.preventDefault();
 }
+class LastDocs extends React.Component {
+    constructor(props) {
```
Comment (Ditmar) [inline src/components/DashboardComponent/Orders.jsx]: We are using react with hooks , no React with classes

**Review by Ditmar — CHANGES_REQUESTED** [review]
don't use classes use hooks

**Comment by RojoRX** [comment]
Good afternoon engineer. Fixed the changes, forgot I was using Orders.jsx as a test at the start. Now all my code is in the LastDocsComponent with the requested changes and I removed my code from Orders.jsx

File: src/components/LastDocsComponent/services/call.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+const apiData = async (state) => {
+    const response = await fetch("https://fakestoreapi.com/products?limit=5");
+    state(await response.json())
```
Comment (Ditmar) [inline src/components/LastDocsComponent/services/call.js:3]: nice

**Review by Ditmar — APPROVED** [review]
approved

**Pull request merged**